foam_controlwrapper/foam_files.py

```diff
@@ -1,61 +1,91 @@
 """ foam_files module
 
-Module for writing OpenFOAM initial files
+Module for writing and modifying OpenFOAM files
 
 """
 import os
 from foam_templates import head, dictionaryTemplates
 from foam_templates import scalarTemplates, vectorTemplates
+from simphony.core.cuba import CUBA
+from cuba_extension import CUBAExt
+from PyFoam.RunDictionary.ParsedParameterFile import ParsedParameterFile
+from PyFoam.Basics.DataStructures import Vector
 
 
 class FoamFiles():
+    """ Module for OpenFOAM file management
```

Comment:
In python unlike other languages like functions can exist outside a class. The FoamFiles class does not look like it holds any state and thus the methods in this class do not really need to be part of a class. I would suggest to remove th class boilerplate and define the methods as normal module level functions.

Generic note about python classes:
- New style vs old style classes, `class FoamFiles():` creates an old style class which is not advised, a more appropriate definition would `class FoamFiles(object):`. The difference between the two definitions are subtle but cann effect operation is unexpected ways.
- If the `__init__` does not do anything there is no reason do define an empty `__init__.py`.  There are some exceptions where it is necessary to override the default constructor but it is not the case here.